src/index.ts

```diff
@@ -278,7 +279,7 @@ export async function install(
           },
         },
         rollupPluginNodeResolve({
-          mainFields: ['module', 'jsnext:main', 'browser', !isStrict && 'main'].filter(Boolean),
+          mainFields: ['browser:module', 'module', 'browser', !isStrict && 'main'].filter(Boolean),
```

Comment:
@dangodev just pinged you on this one because I removed the 'js:next' entrypoint that you added. I think simplicity is better here, given the "point to the exact file" workaround we already have for non-standard packages. lmk if you have a prob with that

Reply:
WFM. I’ve personally only noticed it for Preact packages, and old ones at that. It seems non-standard now. Fine with ignoring it.